kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file defines the API for the core Kolibri app.
+ */
+
+// module must be specified, requireName is optional.
+// module can be used alone to map a Kolibri module into the API.
+// requireName can be used additionally to expose this module to requires in other plugins.
+// By default any module specified with both and included with a relative (rather than absolute)
+// path will be aliased to allow short hand references. Obviously, this is not needed for
+// modules already in node_modules.
+//
+// N.B. You cannot use keys that require quotation marks in this object.
+// e.g. 'content-icon' (although this can be used as a value in requireName or module).
+
+const keys = [
+  'module', // Require statement for the module.
+  'requireName', // Use to indicate the name this module can be 'required' as from other plugins.
+];
+
+const apiSpec = {
+  lib: {
+    logging: {
+      requireName: 'logging',
+      module: require('../logging'),
+    },
+    vue: {
+      requireName: 'vue',
+      module: require('vue'),
+    },
+    vuex: {
+      requireName: 'vuex',
+      module: require('vuex'),
+    },
+    jscookie: {
+      requireName: 'js-cookie',
```

Comment:
This seems to be one of the only cases where the key doesn't match the `requireName` -- is that something that could be collapsed, to simplify the boilerplate here? So it could just be something like: `vuex: require('vuex')`